migrations/postgres/002_create_vendors.up.sql

```diff
@@ -0,0 +1,34 @@
+-- Vendors table
+CREATE TABLE vendors (
+    vendor_id SERIAL PRIMARY KEY,
+    phone_number VARCHAR(20) UNIQUE NOT NULL,
+    email VARCHAR(255) UNIQUE,
+    business_name VARCHAR(255) NOT NULL,
+    contact_person_name VARCHAR(200),
+    phone_verified BOOLEAN DEFAULT false,
+    phone_verification_code VARCHAR(6),
+    phone_verification_expires_at TIMESTAMP,
+    tax_id VARCHAR(50),
```

Comment:
break up into 2 fields: pan_number and gst_number

Reply:
kept `tax_id` as is and added `gst_number`